config/oystehr/zambdas.json

```diff
@@ -1462,6 +1474,18 @@
           }
         ]
       }
+    },
+    "SUB_PATIENT_PAYMENT_CANDID_SYNC_AND_RECEIPT_SUBSCRIPTION": {
```

Comment:
is this missing a required extension?
```
      "extension": [
          {
            "url": "http://zapehr.com/fhir/extension/SubscriptionTriggerEvent",
            "valueString": "create"
          }
        ]
```

Reply:
there are entries in the zambdas file with both extensions. maybe that's better?